sklearn/preprocessing/tests/test_data.py

```diff
@@ -2651,3 +2652,145 @@ def test_minmax_scaler_clip(feature_range):
         X_transformed,
         [[feature_range[0], feature_range[0],
           feature_range[1], feature_range[1]]])
+
+
+def test_spline_transformer_input_validation():
+    # test that we raise errors for invalid input
```

Comment:
In the future, I want to move toward having a check that makes sure that all test have a comment describing it. Using a docstring helps with this:

```suggestion
    """Test that we raise errors for invalid input."""
```

Reply:
Done for all B-spline tests.